modules/installation-vsphere-zone-enablement.adoc

```diff
@@ -0,0 +1,160 @@
+// Module included in the following assemblies:
+//
+//* installing-vsphere-installer-provisioned-customizations.adoc
+//* installing-vsphere-installer-provisioned-network-customizations.adoc
+//* installing_vsphere/installing-vsphere.adoc
+//* installing-vsphere-network-customizations.adoc
+//* installing-restricted-networks-installer-provisioned-vsphere
+//* installing-restricted-networks-vsphere.adoc
+
+:_content-type: CONCEPT
+[id="installation-vsphere-zone-enablement_{context}"]
+= VMware vSphere zone enablement
+
+You can install an {product-title} cluster in a vSphere environment that consists of multiple datacentres running in a single VMware vCenter. 
+
+By specifying multiple datacentres to run in a single VMware vCenter, you can reduce the risk of a hardware failure or network outage in a given failure domain.
+
+You can also specify region and zone information for an {product-title} cluster that you created from a previous release, so that the cluster can run in a zone-enablement environment on vSphere. 
```

Comment:
```suggestion
You can configure multiple regions and zones on a cluster after installation. 
```
Reworded for simplicity. @jcpowermac Would customers on 4.12 and earlier need to upgrade their cluster to 4.13 before being able to configure regions/zones/failure domains?

Reply:
Updated.

Reply:
> Reworded for simplicity. @jcpowermac Would customers on 4.12 and earlier need to upgrade their cluster to 4.13 before being able to configure regions/zones/failure domains?

yes